packages/menu/index.tsx

```diff
@@ -0,0 +1,202 @@
+// The MIT License
+//
+// Copyright (c) 2019 Google, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import * as React from 'react';
+import classnames from 'classnames';
+import {MDCMenuFoundation} from '@material/menu/foundation';
+import {MDCMenuAdapter} from '@material/menu/adapter';
+import MenuSurface, {MenuSurfaceProps} from '@material/react-menu-surface';
+import MenuList, {MenuListProps} from './MenuList';
+import MenuListItem from './MenuListItem';
+
+const {cssClasses} = MDCMenuFoundation;
+
+export interface MenuProps extends Exclude<MenuSurfaceProps, 'ref'> {
+  children: React.ReactElement<MenuList>;
+  handleSelected?: (index: number, item: Element) => void;
```

Comment:
shouldn't the prop name be `onSelected`? just like `onChange` for an `input` element

Reply:
Ya that makes sense. However we use the word `handle` in a few other places. We should normalize that to `on` at some point.